app/services/find/cycle_timetable.rb

```diff
@@ -25,33 +25,30 @@ class CycleTimetable
 
         first_deadline_banner: Time.zone.local(2023, 8, 1, 9), # 5 weeks before Apply 1 deadline
         apply_1_deadline: Time.zone.local(2023, 9, 5, 18), # First Tuesday of September
-        apply_2_deadline: Time.zone.local(2023, 9, 19, 18), # 2 weeks after Apply 1 deadline
+        apply_deadline: Time.zone.local(2023, 9, 19, 18), # 2 weeks after Apply 1 deadline
         find_closes: Time.zone.local(2023, 10, 2, 23, 59, 59) # The evening before Find opens in the new cycle
       },
       2024 => {
         find_opens: Time.zone.local(2023, 10, 3, 9), # First Tuesday of October
         apply_opens: Time.zone.local(2023, 10, 10, 9), # Second Tuesday of October
-        first_deadline_banner: Time.zone.local(2024, 7, 30, 9), # 5 weeks before Apply 1 deadline
-        apply_1_deadline: Time.zone.local(2024, 9, 3, 18), # First Tuesday of September
-        apply_2_deadline: Time.zone.local(2024, 9, 17, 18), # 2 weeks after Apply 1 deadline
+        first_deadline_banner: Time.zone.local(2024, 7, 30, 9),
```

Comment:
Not sure if it matters, but our first deadline banner is going up on 1 July

Reply:
I think there might be different timings for the Apply banners and Find banners. I guess another thing to add to the to check list.